tests/embedding_test_utils.py

```diff
@@ -176,9 +184,17 @@ def test_default_embeddings_stay_the_same_after_saving_and_loading(self):
 
         if self.is_token_embedding:
             for token_old, token_new in zip(sentence_old, sentence_new):
-                assert (token_old.get_embedding(names_old) == token_new.get_embedding(names_new)).all()
+                assert torch.allclose(
+                    token_old.get_embedding(names_old),
+                    token_new.get_embedding(names_new),
+                    atol=1e-6,
+                )
         if self.is_document_embedding:
-            assert (sentence_old.get_embedding(names_old) == sentence_new.get_embedding(names_new)).all()
+            assert torch.allclose(
+                sentence_old.get_embedding(names_old),
+                sentence_new.get_embedding(names_new),
+                atol=1e-6,
```

Comment:
Can you please revert those changes?

The bug that transformer embeddings weren't the same after reloading are fixed on master now.

Reply:
> Can you please revert those changes?

Just to clarify, do you want me to revert all the changes were I introduce `torch.allclose` or just the ones for `if self.is_document_embedding` branching?

Reply:
yes exactly, the test is intended to ensure that embeddings work the same even when reloading them. Hence, it is expected to have the exact same values, as the same computer architecture is used

Reply:
Done.